packages/legend-transac/src/@types/saga/commands/rankings.ts

```diff
@@ -1,7 +1,12 @@
 /**
  * Different commands related to the "rankings" microservice.
  */
-export const rankingsCommands = {} as const;
+export const rankingsCommands = {
+    /**
```

Comment:
This is incorrect; it’s a single-step saga. The only step (for now) is on the blockchain, and the command is executed on the blockchain domain. You’re missing the `commenceSaga`. Rankings triggers the `commenceSaga` → modify `commence.ts`, add an appropriate title for the saga, and properly type the payload.